aioapns/__init__.py,sha256=0SdTeIudeVqGSxAz3IS_TyinmrhO4OJpTeSKn-lUwgQ,326
aioapns/client.py,sha256=f-TdNv-AeQ348i6GVDB1qG0FaQErU11s4LZmkiY9IF4,2211
aioapns/common.py,sha256=Yqf2eIMaYyAFLGbUb6_mQTbnmoy4KQ-3q4Hnu8H_4_k,2590
aioapns/connection.py,sha256=yGF0M12mah4xz-3Q0XOwIBffVTZOaNuHUf0IH5d__XY,17149
aioapns/exceptions.py,sha256=ElPnrfNMDyh8W5jFSLZxXMkNMLKDMo2npWpTyphAJUY,138
aioapns/logging.py,sha256=BmPKdwiqXf4Dzp0kGOSt2YvKqaM1phIZOoMv481xvms,216
aioapns-2.2.dist-info/LICENSE,sha256=bRr3pI_TEb64dF-91Rf49k-52Mu9hybN0i2-5m_fUcM,11348
aioapns-2.2.dist-info/METADATA,sha256=RtjzjiTwYiMq-iBEtox27EgfQ6OQxr5MoXetcXBqP_Q,3615
aioapns-2.2.dist-info/WHEEL,sha256=2wepM1nk4DS4eFpYrW1TTqPcoGNfHhhO_i5m4cOimbo,92
aioapns-2.2.dist-info/top_level.txt,sha256=DDgg0zmpVJly30Ig6nr0JsBVBng7Gnqr8N-NzM7b6XI,8
aioapns-2.2.dist-info/RECORD,,
